patches/src/flopen.c.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/flopen.c b/src/flopen.c
+index 679445f..027f2e6 100644
+--- a/src/flopen.c
++++ b/src/flopen.c
+@@ -37,6 +37,14 @@
+ #include <unistd.h>
+ 
+ #include <libutil.h>
++#ifdef __MVS__
++#define AT_FDCWD -100
++#define LOCK_SH 1       /* Shared lock.  */
++#define LOCK_EX 2       /* Exclusive lock.  */
++#define LOCK_NB         4       /* or'd with one of the above to prevent
++                                   blocking */
++#define LOCK_UN 8       /* Unlock.  */
```

Comment:
I wonder if you can reuse the macros from zoslib here: https://github.com/ibmruntimes/zoslib/blob/36b82277e71446acc066fab77b43bdf5f8914a82/include/sys/file.h#L21

Reply:
@IgorTodorovskiIBM I get error for 'AT_FDCWD', can I define this in zoslib?

Reply:
AT_FDCWD is used by openat, we don't provide an openat definition (yet). It's probably better to leave AT_FDCWD defined here in libbsd.